101-data-factory-blob-to-sql-copy/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+  {
+    "contentVersion": "1.0.0.0",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#", 
+    "parameters": {
+      "dataFactoryName": {
```

Comment:
we use uniqueString(resourceGroup().id) for other globally unique values (e.g. storatge) and it works well...